.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
```

Comment:
```suggestion
<!-- Please include a summary of the change. Please also include relevant motivation and context. -->
```

Reply:
I can understand the dependecies part, but whats wrong with issues link?

Reply:
Because the "summary" kinda implies it describes any issues fixed, so less text. But I don't mind either way.

Reply:
Agree